loopd/swapclient_server.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/lightninglabs/loop/loopdb"
 	clientrpc "github.com/lightninglabs/loop/looprpc"
 	"github.com/lightninglabs/loop/swap"
-	"github.com/lightninglabs/loop/swapserverrpc"
+	serverrpc "github.com/lightninglabs/loop/swapserverrpc"
```

Comment:
Should we use `looprpc` here to look the same as in the first two commits? Or use `serverrpc` there as well?

Reply:
I thought the server import might get upset with 2x `looprpc` packages and one aliased with `looprpc`, but just tried it out and it's fine. Will update!